projects/ComposeConstraintLayout/app/src/main/java/com/example/constraintlayout/verification/VerifyActivity.kt

```diff
@@ -247,6 +254,9 @@ class VerifyActivity : AppCompatActivity() {
 
     @ExperimentalMaterialApi
     fun doNext() {
+        if (composeNum == -1) {
```

Comment:
I am wondering if we should also add the same check for getLayout function  (line 269)

Reply:
It would be redundant